schemas/schema-schema.ipldsch

```diff
@@ -2,7 +2,7 @@
 ## This is the schema describing the schema declarations for IPLD Schemas.
 ## Yes, it's self-describing! :)
 ## -----
-
+##
```

Comment:
Nei, above block does not belong attached to `TypeName`'s docs.

Reply:
Admittedly, it's possible I'm a monster for writing a doc snippet that's not attached to any types or other structure at all.  (I've not done a lot of attending to how comments would be roundtripped through the "AST", should we choose to make a goal of doing so.)

Reply:
yes, you're a monster, I already had to special case this particular line: https://github.com/ipld/js-ipld-schema/blob/e1f80bd8a384d3075f3406b3a4ea222cc18f189e/test/_split-schema-schema.js#L31-L35
otherwise the pattern you established is quite nice